validation.js

```diff
@@ -0,0 +1,14 @@
+const formbutton = document.getElementById('form-button');
+const Email = document.getElementById('email');
+const Errormessage = document.getElementById('form-error');
+Errormessage.textContent = 'Message not sent! Please write using only the small letters. Thank you.';
```

Comment:
- [ ] your submission checks the email well. great job adding an eventlistener to the button to check email. unfortunately, an error message doesn't pop up when a capital letter is included in the email. please look fix that.